tests/test_utils.py

```diff
@@ -339,12 +339,18 @@ def test_is_semantic_version(version, is_semantic):
     'uid, expected',
     [
         (audeer.uid(), True),
+        (audeer.uid(short=True), True),
         (audeer.uid(from_string='from string'), True),
+        (audeer.uid(from_string='from string', short=True), True),
```

Comment:
I would also add a test for uppercase letters.

In https://en.wikipedia.org/wiki/Universally_unique_identifier they state:

> [RFC 4122 Section 3](https://tools.ietf.org/html/rfc4122#section-3) requires that the characters be generated in lower case, while being case-insensitive on input.

So, I guess we need something like:

```suggestion
        (audeer.uid(from_string='from string', short=True), True),
        (audeer.uid(from_string='from string', short=True).upper(), True),
```

Reply:
But I have to admit I'm not completely sure about this.

From what we have written in the docstring, we would expect the test should fail as an uppercase UID cannot be created with `audeer.uid()`

Reply:
Ups. I accidentally committed your suggestion :)

Reply:
Where exactly do we say in the docstring that upper case is not supported?

Reply:
No, you are right. We only say they can be generated with `audeer.uid()`, but not that they have to.

Reply:
So I guess it is fine to stay with `True` for uppercase UIDs.